### PR TITLE
Added my name

### PR DESCRIPTION
**Pull request by adityaraute**
## Description

Added name in README file.

### Type of change

- [x] Updated documentation (updated the readme, templates, or other repo files)

## Checklist:

- [x] I have checked to make sure no other pull requests are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Here's the addition in my profile.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ Here you can easily customize your Typing SVG with a live preview.
 - **[Eke Victor](https://github.com/Evavic44)**
 - **[8BitJonny](https://github.com/8BitJonny)**
 - **[Krish](https://github.com/krishdevdb)**
+- **[Aditya Raute](https://github.com/adityraute)**
```
Comment (DenverCoder1) [inline README.md:59]:
```suggestion
- **[Aditya Raute](https://github.com/adityaraute)**
```

**Review by DenverCoder1 — COMMENTED** [review]
:+1: Looks great!

Thanks for contributing and using Readme Typing SVG :smile:

One thing to fix.. It seems there is a typo in the URL.

**Comment by adityaraute** [comment]
Fixed

**Comment by DenverCoder1** [comment]
Looks great!

I got a notification that you commented, it doesn't seem like there was a separate notification for the suggestion being committed.

**Pull request merged**